src/ci.js

```diff
@@ -116,6 +116,21 @@ exports.getVars = function() {
     }
     return result;
   }
+  // Github actions
+  if(env.CI === 'true' && env.GITHUB_ACTIONS === 'true'){
+
+    var branchName = env.GITHUB_REF;
+    // remove prefix if exists
+    if (branchName.indexOf('refs/heads/') === 0) {
```

Comment:
Can you check that `branchName` exists as well? Unsure if this can ever be `undefined`

Reply:
I defaulted value of branch to empty string